spec/system/consumer/checkout/summary_spec.rb

```diff
@@ -355,11 +355,67 @@
         expect(page).to_not have_content("You have an order for this order cycle already.")
       end
     end
+
+    describe "order confirmation page" do
+      let(:completed_order) {
+        create(:order_ready_to_ship, distributor:,
+                                     order_cycle:, user_id: user.id)
+      }
+      let(:payment) { completed_order.payments.first }
+
+      shared_examples "order confirmation page" do |paid_state, paid_amount|
+        it "displays the relevant information" do
+          expect(page).to have_content paid_state.to_s
```

Comment:
I think `to_s` shouldn't be necessary because it's already a string comparison. But it's no big deal :)